src/Components/WebAssembly/WebAssembly.Authentication/src/Interop/dist/.gitignore

```diff
@@ -1,2 +1,3 @@
-*.js
-*.js.map
+**/*.js
```

Comment:
Same as above, can we just gitignore `dist/`?

This is empty too: https://github.com/dotnet/aspnetcore/tree/93a3b43a655af3fc2c0087ec0851f33e8c14d269/src/Components/WebAssembly/WebAssembly.Authentication/src/Interop/dist

Reply:
Same answer as above, JS build outputs are produced in that folder